src/process/run.cr

```diff
@@ -46,12 +53,18 @@ class Process
   # To wait for it to finish, invoke `wait`.
   #
   # By default the process is configured without input, output or error.
-  def initialize(command : String, args = nil, input = false : Stdio, output = false : Stdio, error = false : Stdio)
-    argv = [command.to_unsafe]
-    args.try &.each do |arg|
-      argv << arg.to_unsafe
+  def initialize(command : String, args = nil, env = nil : Env, clear_env = false : Bool, shell = false : Bool, input = false : Stdio, output = false : Stdio, error = false : Stdio, chdir = nil : String?)
```

Comment:
I still find `clear_env` a bit confusing and would prefer `inherit_env`. `run(..., env: {"foo": "bar"}, clear_env: true)` seems a bit confusing, I think `run(..., env: {"foo": "bar"}, inherit_env:  false)` is much clearer.

I also still think `working_directory` would be a lot clearer, I think its usecase is really rare (I never felt the need for something like that anywhere outside bash scripts yet), so I'd much prefer the clarity for rarely used feature over that little bit more typing.